psydac/feec/tests/test_commuting_projections.py

```diff
@@ -153,7 +168,23 @@ def test_3d_commuting_pro_2(Nel, Nq, p, bc, m):
     error = abs((Dfun_proj.coeffs-Dfun_h.coeffs).toarray()).max()
     assert error < 1e-9
 
-#==============================================================================
+    # TODO: test takes too long in 3D
```

Comment:
I believe that this comment line is not needed any longer. Or is this test still slow?

Reply:
Fixed in commit [549777d](https://github.com/pyccel/psydac/pull/347/commits/549777dee71e5903709fed387ad47c38b7f01ca6)